usaspending_api/api_docs/api_documentation/awards/idvs/activity.md

```diff
@@ -0,0 +1,77 @@
+- order:  (optional, default `desc`) The sort order.  Can be `desc` or `asc`.
+- hide_edge_cases: (optional, default: `false`) Chooses to hide awards that have edge cases. Edge cases include awards with negative or 0 `awarded_amount`, and `period_of_performance_potential_end_date`:`null`
+
+### Response (JSON)
+
+```
+{
+    "results": [
+        {
+            "award_id": 9265562,
+            "awarding_agency": "Department of Agriculture",
+            "awarding_agency_id": 100,
+            "generated_unique_award_id": "CONT_AWD_AG3142K100082_1205_AG3142B070014_1205",
+            "period_of_performance_potential_end_date": "2012-03-07 00:00:00",
+            "parent_award_id": 68921269,
+            "parent_generated_unique_award_id": "CONT_IDV_AG3142B070014_1205",
+            "parent_award_piid": "AG3142B070014",
+            "obligated_amount": 677629.96,
+            "awarded_amount": 677629.96,
+            "period_of_performance_start_date": "2010-02-02",
+            "piid": "AG3142K100082",
+            "recipient_name": "TELOS CORPORATION",
+            "recipient_id": "399137dd-a636-75dd-e6af-0bd518e314f1-C",
+            "grandchild": false
+        }
+    ],
+    "page_metadata": {
+        "hasPrevious": false,
+        "next": 2,
+        "hasNext": true,
+        "previous": null,
+        "page": 1
+    }
+}
+```
+
+### Response Fields
+
+- `award_id`: Internal primary key of award.
+- `awarding_agency`: Name of the agency that awarded this award.
+- `awarding_agency_id`: Internal surrogate key of the awarding agency.
+- `generated_unique_award_id`: Natural key of award.
+- `period_of_performance_potential_end_date`: The date after which no additional costs may be incurred.  May be extended.
+- `parent_award_id`: Internal primary key of parent of this award.
+- `parent_generated_unique_award_id`: Natural key of the parent of this award.
+- `parent_award_piid`: Procurement instrument identifier for the parent of this award.
+- `obligated_amount`: The amount of money agreed upon for this award.
+- `awarded_amount`: The amount of money paid for this award.
+- `period_of_performance_start_date`: The date before which no costs may be incurred.
+- `piid`: Procurement instrument identifier for the award.
+- `recipient_name`: Name of the recipient of the award
+- `recipient_id`: Internal hash of the recipient of the award
+- `grandchild`: True if the award is a child of an IDV that is a child of the requested award. False if the award is a direct child.
+- `hasPrevious`: True if there's a previous page.  False if not.
+- `hasNext`: True if there's a next page.  False if not.
+- `previous`: The previous page number.
+- `next`: The next page number.
+- `page`: The current page number.
+
+### Errors
+Possible HTTP Status Codes:
+
+* 200: On success.
+* 400 or 422 for various types of invalid POST data.
```

Comment:
Have we done anything to confirm we really return 422 for invalid POST data? We had an issue on that before and I don't see any tinyshield updates (or equivalent) on this PR

Reply:
Yes, check the tests already in `awards/tests/test_awards_idvs_activity_v2.py`